tests/conftest.py

```diff
@@ -1,3 +1,8 @@
+# workaround for ruff removing fixture imports
+# they look unused because of dependency injection by pytest
+from test_fixtures import *  # noqa
```

Comment:
a bit annoying to work around, relevant issue: https://github.com/astral-sh/ruff/issues/3295